upp-delivery-provisioner/README.md

```diff
@@ -51,7 +51,6 @@ docker run \
     -e "BINARY_WRITER_BUCKET=$BINARY_WRITER_BUCKET" \
     -e "AWS_MONITOR_TEST_UUID=$AWS_MONITOR_TEST_UUID" \
     -e "COCO_MONITOR_TEST_UUID=$COCO_MONITOR_TEST_UUID" \
-    -e "BRIDGING_MESSAGE_QUEUE_PROXY=$BRIDGING_MESSAGE_QUEUE_PROXY" \
```

Comment:
ive somehow pull this line in from my other pr but none of the others... Either way it needs to be removed so its all good :)